### PR TITLE
Swagger changes for Client grouping

### PR DESCRIPTION
**Pull request by bikamani**

### CODE REVIEW

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json [hunk 1]
```diff
@@ -273,7 +273,7 @@
...
                 "description": "Create, update, or delete the identiies of multiple devices from the IoT hub identity registry. A device identity can be specified only once in the list. Different operations (create, update, delete) on different devices are allowed. A maximum of 100 devices can be specified per invocation. For large scale operations, consider using the import feature using blob storage(https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities). For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-                "operationId": "RegistryManager_BulkDeviceCRUD",
+                "operationId": "Devices_BulkDeviceCRUD",
```
Comment (drwill-ms) [inline sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json:276]: We'd never want to expose a method called BulkDevice**CRUD**. What verb can we use instead?

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json [hunk 2]
```diff
@@ -563,7 +563,7 @@
             "get": {
                 "summary": "Gets the status of all import/export jobs in an iot hub",
                 "description": "Gets the status of all import/export jobs in an iot hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-                "operationId": "JobClient_GetImportExportJobs",
+                "operationId": "Jobs_GetImportExportJobs",
```
Comment (drwill-ms) [inline sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json:566]: We decided the type name *Client (e.g. JobClient) was fine, right? It is just the property name on the main CL client that we'd strip the "Client" suffix.

**Review by drwill-ms — COMMENTED** [review]
Pretty sure Tim cleaned up our swagger edit plan, and moved all edits to the main feature branch into a single file for suggested changes. You'll want to retarget, I think.

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json (hunk 1 above)
Comment (bikamani) [reply sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json:276]: BulkDeviceOpeations? I am open to any suggestions.

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json (hunk 2 above)
Comment (bikamani) [reply sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json:566]: Yes right. This is for generated code, which already adds a suffix Client/RestClient for the first part . Does that answer your question?

**Comment by bikamani** [comment]
> Pretty sure Tim cleaned up our swagger edit plan, and moved all edits to the main feature branch into a single file for suggested changes. You'll want to retarget, I think.

opened up a new PR. Thanks for review and I will address your comments.

**Comment by bikamani** [comment]
Closing PR as not needed anymore

**Pull request closed without merging**